convert_model.py

```diff
@@ -0,0 +1,59 @@
+
```

Comment:
let's try and consolidate all of these files as much as possible. 
Put the convert model into scripts/ with a longer & more descriptive path. (I can do the same for my RL models)